code/commands/PlayerCommands.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+using Sandbox;
+
+using TTTReborn.Player;
+
+namespace TTTReborn.Commands
+{
+    public static class PlayerCommands
```

Comment:
indentation of namespace wrong, also remember the empty line in front of the closing namespace bracket.